apps/journeys-admin/src/components/Team/TeamSelect/TeamSelect.tsx

```diff
@@ -13,6 +12,8 @@ import sortBy from 'lodash/sortBy'
 import { ReactElement, useRef, useState } from 'react'
 import { useTranslation } from 'react-i18next'
 
+import UsersProfiles3 from '@core/shared/ui/icons/UsersProfiles3'
```

Comment:
```suggestion
import UsersProfiles3Icon from '@core/shared/ui/icons/UsersProfiles3'
```

just so we can be clear later on in the file what this component actually is